orchestrator/src/cli/storage/aws_s3.rs

```diff
@@ -8,11 +8,10 @@ pub struct AWSS3CliArgs {
     #[arg(long)]
     pub aws_s3: bool,
 
-    /// The name of the S3 bucket.
-    #[arg(env = "MADARA_ORCHESTRATOR_AWS_S3_BUCKET_NAME", long, default_value = Some("mo-bucket"))]
-    pub bucket_name: Option<String>,
-
-    /// The S3 Bucket Location Constraint.
-    #[arg(env = "MADARA_ORCHESTRATOR_AWS_BUCKET_LOCATION_CONSTRAINT", long)]
-    pub bucket_location_constraint: Option<String>,
+    /// The ARN / Name of the S3 bucket.
+    /// ARN: arn:aws:s3:::name
+    /// We don't need to provide the region and accountID in s3
+    /// because s3 is unique globally.
+    #[arg(env = "MADARA_ORCHESTRATOR_AWS_S3_BUCKET_IDENTIFIER", long, default_value = Some("mo-bucket"))]
+    pub bucket_identifier: Option<String>,
```

Comment:
```suggestion
    #[arg(env = "MADARA_ORCHESTRATOR_AWS_S3_BUCKET_IDENTIFIER", long, default_value = "bucket")]
    pub bucket_identifier: String,
```
I dont find a reason to have this field as option, can you help me understand this

Reply:
Same as above: 
we are following this structure because of clap,
we can take up refactoring clap in next few PRs.
I believe this was discussed offline.